flooder/src/main.rs

```diff
@@ -55,8 +57,10 @@ fn main() -> Result<(), anyhow::Error> {
     let accounts_and_nonces = derive_user_accounts(
         connection.clone(),
         account.clone(),
-        total_users as usize,
+        first_account_in_range,
+        total_users,
         transfer_amount,
+        initialize_accounts,
     );
```

Comment:
I think this can be spliited into such a flow:
```rust
    let accounts_and_nonces = derive_user_accounts(...);
    if !skip_initialization {
        initialize_accounts(&accounts_and_nonces, ...);
    }
```